tests/unit/oplogPopulator/Connector.js

```diff
@@ -40,20 +41,25 @@ describe('Connector', () => {
         it('Should spawn connector with correct pipeline', async () => {
             const createStub = sinon.stub(connector._kafkaConnect, 'createConnector')
                 .resolves();
+            assert.strictEqual(connector.isRunning, false);
             await connector.spawn();
             assert(createStub.calledOnceWith({
                 name: 'example-connector',
                 config: connectorConfig
             }));
+            assert.strictEqual(connector.isRunning, true);
         });
     });
 
     describe('destroy', () => {
         it('Should destroy connector', async () => {
             const deleteStub = sinon.stub(connector._kafkaConnect, 'deleteConnector')
                 .resolves();
+            connector._isRunning = true;
+            assert.strictEqual(connector.isRunning, true);
             await connector.destroy();
```

Comment:
what is the point of this assert?

Reply:
makes sure the getter works properly

Reply:
oh yes, i missed the missing underscore and getter...